.github/workflows/CI.yml

```diff
@@ -13,24 +13,11 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - '2.6'
```

Comment:
ActiveRecord 6 and 7 have different arity requirements for the used #build_arel method. This difference causes the issue and it's quite complex to fix.
At the same time, we need this fixed version only for our current Rails 6 (ActiveRecord 6.1.*) + ruby 2.7.0 solution. 
So, I decided that we could keep our forked version to maintain only the behavior required for our case.